### PR TITLE
Create job run and get

### PR DESCRIPTION
**Pull request by sshah-wework**
Add explicit test to create / get the job run when run args are non-null

### CODE REVIEW

File: src/test/java/marquez/core/services/JobServiceIntegrationTest.java [hunk 1]
```diff
@@ -122,6 +123,18 @@ public void testCreateJobRun_NullArgs() throws UnexpectedException {
...
+  public void testCreateJobRun_NonNullArgs() throws UnexpectedException {
+    Job job = Generator.genJob(namespaceID);
+    String argsJson = "{'foo': 1}";
+    jobService.createJob(namespaceName, job);
+    JobRun jobRun = jobService.createJobRun(namespaceName, job.getName(), argsJson, null, null);
+    Optional<JobRun> jobRunFound = jobService.getJobRun(jobRun.getGuid());
+    assertTrue(jobRunFound.isPresent());
+    assertNotNull(jobRun.getRunArgsHexDigest());
+    assertEquals(argsJson, jobRun.getRunArgs());
+  }
```
Comment (wslulciuc) [inline src/test/java/marquez/core/services/JobServiceIntegrationTest.java]: 💯

**Comment by wslulciuc** [comment]
LGTM
